data-pipeline/src/data_pipeline/datasets/clinvar.py

```diff
@@ -41,21 +40,26 @@ class SkipVariant(Exception):
     pass
 
 
+def escape_xpath_value(value):
```

Comment:
kind of surprised there's not a way in the library to do this, seems like it would come up